plugins/multiverse

```diff
@@ -0,0 +1,4 @@
+repository=https://github.com/Talados/Multiverse.git
+commit=c5285467f0ebcc8f0aaee57911e003e37c30edb1
+warning=All messages which are being sent or received by this plugin go through a 3rd party which is not controlled or verified by the RuneLite Developers.
+authors=Hai Ke
```

Comment:
Who is this? There's no GitHub user by that name. (that's what this field is for, not for a display name, [which you've already set in your `runelite-plugin.properties` file](https://github.com/Talados/Multiverse/blob/c5285467f0ebcc8f0aaee57911e003e37c30edb1/runelite-plugin.properties#L2))

Reply:
This feedback is still pending fyi